SynZip.pas

```diff
@@ -373,12 +373,13 @@   ESynZipException = class(Exception);
         libz='zlib-64.dll'; // as available in \fpc-win64 sub-folder
         {$endif}
       {$endif MSWINDOWS}
-      {$ifdef LINUX} // dynamically linked with new 64-bit TZStream
+      {$ifdef UNIX} // dynamically linked with new 64-bit TZStream

```

Comment:
will it continue to work on Linux / BSD too ?

Reply:
Yes !
When Linux is defined, Unix is also defined.
When Darwin (BSD) is defined, also Unix is defined.